File: 7.Multi Language Basics/swift/basics.swift
// 1. Output
print("Hello from Swift!")

// 2. Variables
// var is mutable, let is immutable (constant)
var score = 85
let average = 78.5

// 3. Conditionals (no parentheses around condition)
if score >= 80 {
    print("Result: Excellent!")
} else {
    print("Result: Try harder!")
}

// 4. Loops
print("For Loop output: ", terminator: "")
for i in 1...5 { // 1 to 5 inclusive range
    print("\(i) ", terminator: "")
}
print()

File: 7.Multi Language Basics/rust/basics.rs
fn main() {
    // 1. Output
    println!("Hello from Rust!");

    // 2. Variables
    // By default, variables in Rust are immutable. Use 'mut' to make them mutable.
    let score = 85;
    let _average = 78.5; // Prefix with underscore to prevent unused variable warnings

    // 3. Conditionals (no parentheses around condition)
    if score >= 80 {
        println!("Result: Excellent!");
    } else {
        println!("Result: Try harder!");
    }

    // 4. Loops
    print!("For Loop output: ");
    for i in 1..=5 { // 1 to 5 inclusive
        print!("{} ", i);
    }
    println!();
}
